extensions/wrapper/clients/typescript-client/src/utils/HttpUtils.ts

```diff
@@ -0,0 +1,9 @@
+export function needsAuthentication(httpStatus: number) {
```

Comment:
Comment: I had to look up what this actually does, what do you think about `is401Or403`? 😄

Reply:
I kept the name but added a docstring that gives you the context